packages/ckeditor5-bookmark/src/ui/bookmarkview.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module bookmark/ui/bookmarkview
+ */
+
+import {
+	View,
+	FormHeaderView,
+	CssTransitionDisablerMixin,
+	type ViewWithCssTransitionDisabler,
+	type ViewCollection
+} from 'ckeditor5/src/ui.js';
+import {
+	type Locale
+} from 'ckeditor5/src/utils.js';
+import BookmarkFormView, { type BookmarkFormValidatorCallback } from './bookmarkformview.js';
+
+import '../../theme/bookmark.css';
+
+/**
+ * The bookmark form view controller class.
+ *
+ * See {@link module:bookmark/ui/bookmarkformview~BookmarkFormView}.
+ */
+export default class BookmarkView extends View {
+	/**
+	 * A collection of form child views in the form.
+	 */
+	public readonly children: ViewCollection;
+
+	/**
+	 * The form view displayed inside the balloon.
+	 */
+	public formView: BookmarkFormView & ViewWithCssTransitionDisabler;
+
+	/**
+	 * The bookmark header inside the balloon.
+	 */
+	private _formHeader: FormHeaderView;
+
+	/**
+	 * Creates an instance of the {@link module:bookmark/ui/bookmarkformview~BookmarkFormView} class.
+	 *
+	 * Also see {@link #render}.
+	 *
+	 * @param locale The localization services instance.
+	 * @param validators Form validators used by {@link #isValid}.
+	 */
+	constructor( locale: Locale, validators: Array<BookmarkFormValidatorCallback> ) {
+		super( locale );
+
+		this.children = this.createCollection();
+
+		this.formView = new ( CssTransitionDisablerMixin( BookmarkFormView ) )( locale, validators );
+		this._formHeader = new FormHeaderView( locale, {
+			label: this.t!( 'Bookmark' )
+		} );
+
+		const classList = [ 'ck', 'ck-bookmark-view' ];
+
+		this.children.add( this._formHeader );
```

Comment:
Maybe we could we drop the private `_formHeader` property and just add it to the children collection directly?

```suggestion
		this.children.add( new FormHeaderView( locale, {
			label: this.t!( 'Bookmark' )
		} ) );
```

I've seen that this approach is mostly used in your codebase. For example: https://github.com/ckeditor/ckeditor5/blob/23a35bd30914544152c39ef6f6459e3eccedc2fe/packages/ckeditor5-table/src/tableproperties/ui/tablepropertiesview.ts#L268-L271

Reply:
I agree it make sense, but for testing purpose it is much more convenient to have reference for that view component, otherwise we need to provide `locale` and `t()` so it requires creating the instance of editor.